<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>951</width>
    <height>731</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Communication</string>
  </property>
  <widget class="QTabWidget" name="TypeTab">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>130</y>
     <width>221</width>
     <height>251</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="serialTab">
    <attribute name="title">
     <string>Serial</string>
    </attribute>
    <widget class="QWidget" name="formLayoutWidget">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>201</width>
       <height>135</height>
      </rect>
     </property>
     <layout class="QFormLayout" name="serForm">
      <property name="labelAlignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
      <property name="formAlignment">
       <set>Qt::AlignRight|Qt::AlignTop|Qt::AlignTrailing</set>
      </property>
      <property name="rightMargin">
       <number>7</number>
      </property>
      <item row="0" column="0">
       <widget class="QLabel" name="l_comport">
        <property name="text">
         <string>COM Port</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QComboBox" name="port_box">
        <item>
         <property name="text">
          <string>COM1</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>COM2</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>COM3</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>COM4</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>COM5</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>COM6</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>COM7</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>COM8</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>COM9</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>COM10</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="l_baudrate">
        <property name="text">
         <string>Baud Rate</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QComboBox" name="speed_box">
        <item>
         <property name="text">
          <string>9600</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>19200</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>38400</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="l_parity">
        <property name="text">
         <string>Parity</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QComboBox" name="parity_box">
        <item>
         <property name="text">
          <string>NONE</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>EVEN</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>ODD</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="l_stopbits">
        <property name="text">
         <string>Stop Bits</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QComboBox" name="stopbit_box">
        <item>
         <property name="text">
          <string>1</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>2</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="QLabel" name="l_databit">
        <property name="text">
         <string>Data Bits</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item row="4" column="1">
       <widget class="QComboBox" name="databit_box">
        <item>
         <property name="text">
          <string>7</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>8</string>
         </property>
        </item>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>150</y>
       <width>201</width>
       <height>23</height>
      </rect>
     </property>
     <layout class="QFormLayout" name="serdtForm">
      <property name="rightMargin">
       <number>7</number>
      </property>
      <item row="0" column="0">
       <widget class="QLabel" name="l_dt1">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>Delay Time (ms)</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="dtTxt1">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QPushButton" name="serBtn">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>191</y>
       <width>86</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>Connect</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="etherTab">
    <attribute name="title">
     <string>Ethernet</string>
    </attribute>
    <widget class="QWidget" name="formLayoutWidget_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>201</width>
       <height>131</height>
      </rect>
     </property>
     <layout class="QFormLayout" name="formLayout_2">
      <property name="labelAlignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
      <property name="formAlignment">
       <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
      </property>
      <property name="rightMargin">
       <number>7</number>
      </property>
      <item row="0" column="0">
       <widget class="QLabel" name="l_ip">
        <property name="text">
         <string>　IP</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="l_port">
        <property name="text">
         <string>　Port</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="ipTxt"/>
      </item>
      <item row="1" column="1">
       <widget class="QLineEdit" name="portTxt"/>
      </item>
     </layout>
    </widget>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>150</y>
       <width>201</width>
       <height>23</height>
      </rect>
     </property>
     <layout class="QFormLayout" name="etherForm">
      <property name="rightMargin">
       <number>7</number>
      </property>
      <item row="0" column="0">
       <widget class="QLabel" name="l_dt2">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>Delay Time (ms)</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="dtTxt2">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QPushButton" name="etherBtn">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>191</y>
       <width>86</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>Connect</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QGroupBox" name="TypeBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>221</width>
     <height>111</height>
    </rect>
   </property>
   <property name="title">
    <string>Communication Type</string>
   </property>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>20</y>
      <width>133</width>
      <height>80</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <property name="topMargin">
      <number>5</number>
     </property>
     <item>
      <widget class="QRadioButton" name="rtuBtn">
       <property name="text">
        <string>MODBUS RTU</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="asciiBtn">
       <property name="text">
        <string>MODBUS ASCII</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="tcpBtn">
       <property name="text">
        <string>MODBUS TCP</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QGroupBox" name="codeBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>390</y>
     <width>221</width>
     <height>141</height>
    </rect>
   </property>
   <property name="title">
    <string>Reserved Command</string>
   </property>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>20</y>
      <width>113</width>
      <height>31</height>
     </rect>
    </property>
    <layout class="QFormLayout" name="formLayout_5">
     <property name="formAlignment">
      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
     </property>
     <item row="0" column="0">
      <widget class="QLabel" name="l_addr">
       <property name="text">
        <string>Address</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QSpinBox" name="addrSpin">
       <property name="maximum">
        <number>255</number>
       </property>
       <property name="value">
        <number>1</number>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>70</y>
      <width>91</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Fuction Code</string>
    </property>
   </widget>
   <widget class="QWidget" name="">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>90</y>
      <width>141</width>
      <height>30</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="fc03">
       <property name="toolTip">
        <string/>
       </property>
       <property name="whatsThis">
        <string/>
       </property>
       <property name="text">
        <string>03</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="fc06">
       <property name="text">
        <string>06</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="fc16">
       <property name="text">
        <string>16</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QGroupBox" name="CommBox">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>280</y>
     <width>681</width>
     <height>441</height>
    </rect>
   </property>
   <property name="title">
    <string>Communtication Data</string>
   </property>
   <widget class="QTextBrowser" name="TxText">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>661</width>
      <height>181</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextBrowser" name="RxText">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>250</y>
      <width>661</width>
      <height>181</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="TxLabel">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>131</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Tx Window </string>
    </property>
   </widget>
   <widget class="QLabel" name="RxLabel">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>230</y>
      <width>81</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Rx Window</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>10</y>
     <width>681</width>
     <height>261</height>
    </rect>
   </property>
   <property name="title">
    <string>Send Data</string>
   </property>
   <widget class="QFrame" name="LoopFrame">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>671</width>
      <height>251</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QPlainTextEdit" name="loopTxt">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>60</y>
       <width>661</width>
       <height>181</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="loop_inBtn">
     <property name="geometry">
      <rect>
       <x>570</x>
       <y>10</y>
       <width>91</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Input</string>
     </property>
    </widget>
    <widget class="QPushButton" name="stopBtn">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>470</x>
       <y>10</y>
       <width>91</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>STOP</string>
     </property>
    </widget>
   </widget>
   <widget class="QRadioButton" name="singleBtn">
    <property name="geometry">
     <rect>
      <x>23</x>
      <y>25</y>
      <width>101</width>
      <height>19</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="text">
     <string>Single</string>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QRadioButton" name="loopBtn">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>25</y>
      <width>111</width>
      <height>19</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="text">
     <string>Loop</string>
    </property>
   </widget>
   <widget class="QPushButton" name="TxClearBtn">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>20</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Clear</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="saveBtn">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>27</y>
      <width>131</width>
      <height>20</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="text">
     <string>Save File</string>
    </property>
   </widget>
   <widget class="QFrame" name="singleFrame">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>10</y>
      <width>681</width>
      <height>251</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QLineEdit" name="lineTxt">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>60</y>
       <width>661</width>
       <height>181</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="single_inBtn">
     <property name="geometry">
      <rect>
       <x>480</x>
       <y>10</y>
       <width>191</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Input</string>
     </property>
    </widget>
   </widget>
   <zorder>LoopFrame</zorder>
   <zorder>singleFrame</zorder>
   <zorder>saveBtn</zorder>
   <zorder>singleBtn</zorder>
   <zorder>TxClearBtn</zorder>
   <zorder>loopBtn</zorder>
  </widget>
  <widget class="QGroupBox" name="langBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>540</y>
     <width>221</width>
     <height>71</height>
    </rect>
   </property>
   <property name="title">
    <string>Language</string>
   </property>
   <widget class="QRadioButton" name="korBtn">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>31</y>
      <width>91</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Korean</string>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QRadioButton" name="engBtn">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>31</y>
      <width>91</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>English</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
